src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2678,6 +2679,7 @@ namespace wallet_rpc
         KV_SERIALIZE(ssl_ca_file)
         KV_SERIALIZE(ssl_allowed_fingerprints)
         KV_SERIALIZE_OPT(ssl_allow_any_cert, false)
+        KV_SERIALIZE_OPT(is_connected, false)
```

Comment:
This  is up for debate, as it *may* break people's expectations of this RPC call, but I think that the connection checking should default to `true`. This depends on how most people use this function, but I think it makes the most sense to fail if the daemon you try setting can't be connected to, even if updating just the information itself succeeded. Discerning between the 2 scenarios can be left up to error codes.

Reply:
Since the wallet rpc will fail fast if a user attempts to start with no daemon running this extension of the rpc would most likely be useful after the fact. Enter in the last known good address with this flag to ensure it is still online. My mindset was that the bug is missing functionality / error code so I left it as false in order to maintain backwards compatibility. Let's see what others think as well. @woodser do you think this will suffice for #8314? I know it was a request for a separate addition to wallet rpc but I thought this might be an opportunity to kill to birds with one one commit.